### PR TITLE
Feature/help texts

### PR DESCRIPTION
**Pull request by Roemer**
- Added a new Tooltip component (so the tooltips are always the same)
- Added more help texts
- Fixed build and notes

Check especially the new help texts if they contain the correct information.

Replaces PR #247

### CODE REVIEW

File: src/components/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Tooltip as MuiTooltip } from "@material-ui/core";
+import React from "react";
+
+export class Tooltip extends React.Component<any, any> {
```
Comment (pashidlos) [inline src/components/Tooltip.tsx:4]: make it as functional component 
also avoid using `any`

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 - clone repo
 - `npm i`
+- `npm run build`
```
Comment (pashidlos) [inline README.md:9]: not sure that it's needed to run before local start
why would you need to run it?

**Review by pashidlos — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Roemer) [reply README.md]: Right, works without. I removed it.

File: src/components/Tooltip.tsx (hunk 1 above)
Comment (Roemer) [reply src/components/Tooltip.tsx]: Ok, I tried to make it a functional component, seems to work.

**Review by pashidlos — APPROVED** [review]
LGTM!

**Pull request merged**